### PR TITLE
.github/advanced-issue-labeler.yml: Add labels for VP3200 platforms

### PR DESCRIPTION
**Pull request by SebastianCzapla**

### CODE REVIEW

**Comment by SebastianCzapla** [comment]
White space changes appear even if just one line is changed, not sure why this is happening.

**Review by JanPrusinowski — APPROVED** [review]

File: .github/advanced-issue-labeler.yml [hunk 1]
```diff
@@ -67,7 +67,7 @@ policy:
             keys: ['Protectli VP4630', 'Protectli VP4650', 'Protectli VP4670']
 
           - name: protectli_vault_adl
-            keys: ['Protectli VP6630', 'Protectli VP6650', 'Protectli VP6670']
+            keys: ['Protectli VP6630', 'Protectli VP6650', 'Protectli VP6670', 'Protectli VP3210', 'Protectli VP3230']
```
Comment (macpijan) [inline .github/advanced-issue-labeler.yml:70]: @SebastianCzapla The VP3000 series is much different from the VP6000 series. It is still Alder Lake, but the VP3000 is ADL-N. We likely need a new label in the same way as we have in the source code: `vault_adln`

**Review by macpijan — CHANGES_REQUESTED** [review]

File: .github/advanced-issue-labeler.yml (hunk 1 above)
Comment (SebastianCzapla) [reply .github/advanced-issue-labeler.yml]: Fixed

**Comment by SebastianCzapla** [comment]
Dropdown device list also updated

**Review by macpijan — APPROVED** [review]
LGTM, let's see if it works.

**Comment by macpijan** [comment]
Works: https://github.com/Dasharo/dasharo-issues/issues/1103

**Pull request merged**